.github/workflows/stale.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  invalid-to-stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is invalid or waiting response because it has been open 7 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
```

Comment:
This issue is marked as `stale` because it has been marked as `invalid` or `awaiting response` for 7 days without any further response. It will be closed in 5 days if the `stale` label is not removed or if there is no further response.